include/dem/dem.h

```diff
@@ -154,6 +156,12 @@ class DEMSolver
   inline bool
   check_contact_search_step_dynamic();
 
+  /**
+   * @brief Establish if this is a load balance step using the adequate method.
```

Comment:
This is defined first and then is_contact_search_step(). I would maybe put contact search below load balance as it is in dem.cc